.github/workflows/format.yml

```
@@ -0,0 +1,26 @@
+name: "Format Code"
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+      - name: Install dependencies
+        run: npm ci
+      - name: npm format
+      - run: npm format
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "[ci] npm format"
+          branch: ${{ github.head_ref }}
```

Not sure how crucial this is, but elsewhere we specify FredKBot as the committer:

```suggestion
          branch: ${{ github.head_ref }}
          commit_user_name: fredkbot
          commit_user_email: fred+astrobot@astro.build
```

We may need to add the token to the repo secrets. @FredKSchott is probably the only one with permissions to do that.

---

hmmm okay; not the case in prettier-plugin-astro ig

---

This only really matters on the main repo, where the main branch is protected

---

so what's the verdict? I can probably just leave it as is, right?

---

This is a conspiracy to keep FredKBot’s GitHub stats down!!! 😡